pkg/apis/configs/authorization/defaults.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authorization
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Defaults includes the default values to be populated by the webhook.
+type Defaults struct {
+	// NamespaceDefaults are the default Authorizations CRDs for each namespace. namespace is the
+	// key, the value is the default AuthorizationTemplate to use.
+	NamespaceDefaults map[string]ScopedDefaults `json:"namespaceDefaults,omitempty"`
+	// ClusterDefaultAuthorization is the default Authorization CRD for all namespaces that are not in
+	// NamespaceDefaultAuthorizations.
+	ClusterDefault ScopedDefaults `json:"clusterDefaults,omitempty"`
+}
+
+type ScopedDefaults struct {
```

Comment:
might be useful to add some of comments from the config map to each of these attributes here..

Reply:
Done.